docs/integrations/mirroring.md

```diff
@@ -103,13 +104,13 @@ def get_modified_remote_data_command(client, args):
 ```
 
 ### update-remote-system
-* UpdateRemoteSystemArgs - this is an object created to maintain all the arguments you receive from the server in order to use this command.
+* **UpdateRemoteSystemArgs** - this is an object created to maintain all the arguments you receive from the server in order to use this command.
 Arguments explanation:
-  - data - represents the data of the current incident - a dictionary object `{key: value}`.
-  - entries - represents the entries from your current incident - a list of dictionary objects representing the entries.
-  - remote_incident_id - contains the value of the dbotMirrorId field, which represents the id of the incident in the external system. 
-  - inc_status - the status of the incident(numeric value, could be used with IncidentStatus from CommonServerPython).
-  - delta - represents the dictionary of fields which have changed from the last update - a dictionary object `{key: value}` containing only the changed fields.
+  - *data* - represents the data of the current incident - a dictionary object `{key: value}`.
+  - *entries* - represents the entries from your current incident - a list of dictionary objects representing the entries.
+  - *remote_incident_id* - contains the value of the dbotMirrorId field, which represents the id of the incident in the external system. 
+  - *inc_status* - the status of the incident(numeric value, could be used with IncidentStatus from CommonServerPython).
+  - *delta* - represents the dictionary of fields which have changed from the last update - a dictionary object `{key: value}` containing only the changed fields.
```

Comment:
Another parameter is `incidentChanged` which is boolean to tell us if the incident itself changed. Because incident might not change but we want to send mirrored entries over.